.github/workflows/publish_alpha.yml

```
@@ -0,0 +1,137 @@
+          token: ${{ secrets.GITHUB_TOKEN }}
+          maxIssues: 50
+        id: changelog
+
+      - name: Commit to dev
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: Increment Version
+          branch: dev
+
+      - name: version
+        run: echo "::set-output name=version::$(python setup.py --version)"
```

actionlint

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)